format/format.sh

```diff
@@ -58,13 +59,15 @@ case "$mode" in
    blackmode="--check"
    javamode="--set-exit-if-changed --dry-run"
    gofmtmode="-l"
+   bufmode="format -d"
```

Comment:
from the docs, looks like you also have to ask for a non-zero exit code so this will properly fail on CI when formatting is needed